dashboard/app/helpers/aichat_safety_helper.rb

```diff
@@ -0,0 +1,57 @@
+# Provides functionality to detect toxicity in user input and model output used in the AI Chat Lab.
+# Uses various services to check for profanity and toxicity based on DCDO settings.
+module AichatSafetyHelper
+  class ToxicityDetector
+    DEFAULT_TOXICITY_THRESHOLD_USER_INPUT = 0.2
```

Comment:
I think Sam would like input threshold to be `0.3` and output to be `0.5` https://codedotorg.slack.com/archives/C07LQF3AGMA/p1726238288468869

Reply:
Updated